index.html

```diff
@@ -0,0 +1,153 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <title>Awesome</title>
+    <link rel="stylesheet" href="reset.css">
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <section class="tabs">
+        <div class="tabs__tabs-container">
+          <p class="tabs__tab-title">TAB 1</p>
```

Comment:
You don't have cursor changing on hover because you haven't used here button element.